act-rules-format.bs

```diff
@@ -17,7 +17,7 @@ Introduction {#intro}
 
 There are currently many test procedures and tools available which aid their users in testing web content for conformance to accessibility standards such as the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [[WCAG]]. As the web develops in both size and complexity, these procedures and tools are essential for managing the accessibility of resources available on the web.
 
-This format is intended to enable a consistent interpretation of how to test for [accessibility requirements](#accessibility-requirements) and promote consistent results of accessibility tests. It is intended to be used to describe both manual accessibility tests as well as automated tests performed by accessibility testing tools.
+This format is intended to enable a consistent interpretation of how to test [=accessibility requirements documents=] and promote consistent results of accessibility tests. It is intended to be used to describe both manual accessibility tests as well as automated tests performed by accessibility testing tools.
```

Comment:
Should it be "how to create tests for accessibility requirements documents" or "how to test for the requirements in accessibility requirements documents"?
We are not really testing the documents, but the requirements in the documents...

Reply:
Changed it to "how to test for conformance to accessibility requirements documents".